draft-irtf-cfrg-vdaf.md

```diff
@@ -2696,6 +2704,12 @@ def prep_init(Poplar1, verify_key, agg_id, agg_param,
         Poplar1.decode_input_share(input_share)
     Field = Poplar1.Idpf.current_field(level)
 
+    # Ensure that candidate prefixes are all unique and appear in
+    # lexicographic order.
+    for i in range(1,len(prefixes)):
+        if prefixes[i-1] >= prefixes[i]:
+            raise ERR_INPUT # out-of-order prefix
+
```

Comment:
Reviewer note: We might consider adding this check to aggregation parameter encoding as well.